src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputFinalLocalVariableBreakOrReturn.java

```diff
@@ -0,0 +1,61 @@
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+
+public class InputFinalLocalVariableBreakOrReturn {
+
+    void foo1() throws Exception {
+        Exception e; // violation
+        final int a = (int) Math.random();
+        final int b = (int) Math.random();
+
+        switch (a) {
+        case 0:
+            e = new Exception();
+            break;
+        case 1:
+            if (b == 0) {
+                e = new Exception();
+                break;
+            }
+
+            if (b == 0) {
```

Comment:
Emm... I added smth when I was debugging and forgot to change it back. Sorry for my careless